example/ios/Podfile

```diff
@@ -56,5 +56,11 @@ target 'Auth0Example' do
       :mac_catalyst_enabled => false
     )
     __apply_Xcode_12_5_M1_post_install_workaround(installer)
+    
+    installer.pods_project.targets.each do |target|
+      target.build_configurations.each do |config|
+        config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= ['$(inherited)', '_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION']
+      end
+    end
```

Comment:
What is this needed for?

Reply:
Seems to be required to compile since XCode 15 as mentioned [here](https://github.com/facebook/react-native/issues/37748#issuecomment-1580334650)